tests/sanity/ignore-2.18.txt

```diff
@@ -0,0 +1 @@
+tests/unit/mock/loader.py pylint:arguments-renamed
```

Comment:
IMO, this can be and should be fixed like the other collections.

Reply:
None of the ignores in https://github.com/ansible-collections/trendmicro.deepsec/tree/main/tests/sanity is really required.